apps/api-gateway/src/credential-definition/credential-definition.controller.ts

```diff
@@ -38,15 +38,15 @@ export class CredentialDefinitionController {
   @Roles(OrgRoles.OWNER, OrgRoles.ADMIN, OrgRoles.ISSUER, OrgRoles.VERIFIER, OrgRoles.MEMBER)
```

Comment:
Summary and description should not be same

Reply:
Resolved